src/NLog/LayoutRenderers/GuidLayoutRenderer.cs

```diff
@@ -67,13 +67,26 @@ public GuidLayoutRenderer()
         [DefaultValue(false)]
         public bool GeneratedFromLogEvent { get; set; }
 
+        /// <summary>
+        /// Get raw value
+        /// </summary>
+        object IRawValue.GetRawValue(LogEventInfo logEvent) => GetValue(logEvent);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename parameter 'logEvent' to 'logEventInfo' to match the interface declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS927)